### DISCUSSION TITLE
Eul is on the Github Trending page!!!

### DISCUSSION THREAD

**Discussion by jevonmao** [gao-sun/eul#181] [General]
I probably sound like a fool for being so excited about this, @gao-sun and everyone else probably already knew. But anyways, it's really surprising, today I always browsing the Github trending page, and I see this:
Yup. Eul is the first on weekly trending page. 4.7k solid stars.
![Screenshot 2021-02-01 164456](https://user-images.githubusercontent.com/64660730/106536467-60bc3900-64ad-11eb-80c8-59191039ba91.png)

Link source: https://github.com/trending/swift?since=weekly

**Comment by gao-sun** [comment]
this is incredible! sorry i missed this one. thank you for taking the photo of this glorious moment. cheers

**Reply by sweetmans** [reply]
Would I become a contributed? Plan adding icon for ech semen.
